packages/now-cli/scripts/preinstall.js

```diff
@@ -41,6 +47,17 @@ function validateNodeVersion() {
   return { isValid, expected: semver, actual: process.versions.node };
 }
 
+function isInPath(name) {
```

Comment:
I think we need to be careful with this because it might have false positives. Someone recently reported that `now` was printing the current date in Windows Powershell so we don't want to burden the user if this is not truly Now CLI.

Reply:
Ok, I updated to check for `now` inside the global `node_modules` directory instead.